fitbit/api.py

```diff
@@ -456,8 +453,7 @@ def food_goal(self, calories=None, intensity=None, personalized=None):
         """
         Implements the following APIs
 
-        https://wiki.fitbit.com/display/API/API-Get-Food-Goals
-        https://wiki.fitbit.com/display/API/API-Update-Food-Goals
+		https://dev.fitbit.com/docs/food-logging/#get-food-goals
```

Comment:
Remove the extra indent please. And add the update URL
https://dev.fitbit.com/docs/food-logging/#update-food-goal